tests/squash.bats

```diff
@@ -141,3 +143,30 @@ function check_lengths() {
   run_buildah rmi -f testsquash
   run_buildah rmi -f test
 }
+
+# Test build with --squash and --layers and verify number of layers and content inside image
+@test "bud-squash-should-use-cache and verify content inside image" {
+  mkdir -p ${TEST_SCRATCH_DIR}/bud/platform
+
+  cat > ${TEST_SCRATCH_DIR}/bud/platform/Dockerfile << _EOF
+FROM quay.io/libpod/busybox:latest
```

Comment:
```suggestion
FROM busybox
```
The `registries.conf` the test uses sets this up as an alias, so being more specific here doesn't provide a benefit.